designer/client/src/components/Process/ProcessStateIcon.tsx

```diff
@@ -33,45 +29,38 @@ function Errors({ state }: { state: ProcessStateType }) {
     );
 }
 
-function StateIconPopover({
-    processName,
-    processState,
-    tooltip,
-    children,
-}: PropsWithChildren<{
-    processState: ProcessStateType;
-    processName: ProcessId;
-    tooltip: string;
-}>) {
-    const theme = useTheme();
-    const imagePopover = useMemo(() => {
-        const className = css({
-            marginTop: "5px",
-            "&, h3": { backgroundColor: theme.colors?.primaryBackground },
-        });
-        return (
-            <Popover id="state-icon-popover" className={className} title={processName}>
-                <strong>{tooltip}</strong>
-                <Errors state={processState} />
-            </Popover>
-        );
-    }, [processName, processState, theme.colors?.primaryBackground, tooltip]);
-
-    return (
-        <OverlayTrigger trigger="click" placement="left" overlay={imagePopover}>
-            {children}
-        </OverlayTrigger>
-    );
-}
-
 function ProcessStateIcon({ process, processState }: Props) {
     const icon = ProcessStateUtils.getStatusIcon(process, processState);
     const tooltip = ProcessStateUtils.getStatusTooltip(process, processState);
 
+    const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
+
     return (
-        <StateIconPopover processName={process.name} processState={processState} tooltip={tooltip}>
-            <UrlIcon src={icon} title={tooltip} />
-        </StateIconPopover>
+        <>
+            <UrlIcon src={icon} title={tooltip} onClick={(event) => setAnchorEl(event.currentTarget)} />
```

Comment:
It's not the regression because I'm able to reproduce it on the Staging, but on Safari (16.4) The icon is missing. As I see, also metrics icon is missing. Can you take a look at it in this pr, or maybe we should create another task for it? 

<img width="292" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/8eba28a8-7310-4a21-88e7-bde77c9addec">

Reply:
will fix it here

Reply:
no, i'll do separate pr. it's to big to mix with this.